src/vendors/Datadog.Trace.Vendoring/Datadog.Trace.Vendoring.csproj

```
@@ -0,0 +1,61 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <RootNamespace>Datadog.Trace.Vendoring</RootNamespace>
+    <TargetFrameworks Condition="'$(OS)' == 'Windows_NT'">net45;net46;netstandard2.0</TargetFrameworks>
```

This should probably be `net461` instead of `net46` to match `Datadog.Trace` and its NuGet package.
```suggestion
    <TargetFrameworks Condition="'$(OS)' == 'Windows_NT'">net45;net461;netstandard2.0</TargetFrameworks>
```